js/blockchain.js

```diff
@@ -297,15 +89,12 @@ class MobileTwitter {
         if (requestType === 'following') {
             requestType = 'friends'
         }
-        let position = $(".GridTimeline-items").data('min-position');
-        let lastRequestTime = Date.now();
         let cursor = null;
         const _getIDData = () => {
             if (!finderRunning) return false;
             const count = 5000;
             let url = `${requestType}/ids.json?screen_name=${profileUsername}&count=${count}&stringify_ids=true`
             if (cursor) url += `&cursor=${cursor}`
-            lastRequestTime = Date.now();
```

Comment:
Why did you remove these lines (300, 301, 308)?

Reply:
Because both `lastRequestTime` and `position` are not used anywhere, and `$(".GridTimeline-items")` element is not available in current Twitter.